tests/unit/hive_metastore/test_table_migrate.py

```diff
@@ -51,7 +51,7 @@ def test_migrate_dbfs_root_tables_should_produce_proper_queries(ws):
     group_manager = GroupManager(backend, ws, "inventory_database")
     migration_status_refresher = MigrationStatusRefresher(ws, backend, "inventory_database", table_crawler)
     principal_grants = create_autospec(PrincipalACL)
-    table_migrate = TablesMigrator(
+    table_migrator = TablesMigrator(
```

Comment:
renames are outside of scope of this PR, please revert to only necessary changes

Reply:
done

Reply:
Yep, just submitting PRs